src/main/kotlin/net/lumalyte/lg/infrastructure/persistence/guilds/GuildRepositorySQLite.kt

```
@@ -388,6 +396,16 @@ class GuildRepositorySQLite(private val storage: Storage<Database>) : GuildRepos
             null
         }
 
+        val allyHomeAllowedGuilds = try {
+            val csv = rs.getString("ally_home_allowed_guilds").orEmpty()
+            csv.split(",")
+                .filter { it.isNotBlank() }
+                .map { UUID.fromString(it.trim()) }
+                .toSet()
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)